common/countries/PegasianColonies.txt

```diff
@@ -17,58 +17,39 @@ historical_idea_groups = {
 	spy_ideas
 }
 
-historical_units = {
-	western_medieval_infantry
-	chevauchee
-	western_longbow
-	gaelic_mercenary
-	gaelic_free_shooter
-	french_caracolle
-	irish_charge
-	anglofrench_line
-	french_dragoon
-	british_redcoat
-	british_hussar
-	british_square
-	mixed_order_infantry
-	open_order_cavalry
-	napoleonic_square
-	napoleonic_lancers
-}
```

Comment:
I don't get why these were here in the first place. One half of the equestrian states having these lines, while the other half didn't.